certbot-consul/setup.py

```diff
@@ -0,0 +1,22 @@
+"""Setup script for certbot-consul"""
+
+from setuptools import find_packages, setup
+
+version = '1.22.0.dev0'
+
+setup(
+    name='certbot-consul',
+    version=version,
+    packages=find_packages(),
+    python_requires='>=3.6',
+    install_requires=[
+        f'certbot>={version}',
+        'python-consul>=1.1.0',
```

Comment:
Is 1.1.0 really the minimal version required? For example, Debian and Ubuntu seems to be shipping [`python3-consul` version 0.7.1](https://pkgs.org/search/?q=python3-consul) currently. Not sure if CentOS/RHEL even ships a Python consul package. At least I can't find it. Gentoo also doesn't have a package for it.